.ci_scripts/run_build.sh

```diff
@@ -11,8 +11,17 @@ case "$BUILD_TARGET" in
 	cd build && make && make test && make install
 	echo "Creating disk image"
 	hdiutil create -volname Julius -srcfolder julius.app -ov -format UDZO julius.dmg
+	cd ..
 	;;
 *)
 	cd build && make && make test
+	if [ "$BUILD_TARGET" == "appimage" ];
```

Comment:
Create an `appimage` case instead of putting this `if` inside the "catch-all" `*)`. Yes, that's duplication of the `cd build && make && make test` line, but the code is clearer.